client/cz-component-dash.html

```diff
@@ -104,57 +104,74 @@
           type: 'Object',
           value: function() { return []; },
         },
-        component: {
+        updateSLO: {
           type: 'Object',
-          value: function() { return {}; },
+          value: function() { return null; },
```

Comment:
I didn't know we could use values directly, cool. I assume the functions are used for non-primitive value types then.